### ISSUE TITLE
Removing all commands excepted math

### ISSUE THREAD

**Issue by Futyn-Maker** [pkubowicz/opendetex#77]
Hello!
Thanks for this program, it's very useful.

I'd like to know if I can remove all LaTeX commands excepted math formulas.
I mean, I want to remove, for example, `itemize`, `textit` and others, but save `\frac{}{}`, `\sqrt{}`, etc.

Thanks!

**Comment by pauloney** [comment]
Yeah, it is possible, but it demands some work. One simple way to do it
would be to list all commands you want to remove, abd write a script just
to do that. The problems are many including the fact that "itemize" is NOT
a command, but the name of an environment.

Paulo Ney
